tests/data_files/Makefile

```diff
@@ -121,7 +121,30 @@ all_intermediate += test-ca2.req.sha256
 
```

Comment:
I reviewed `gitp range-diff upstream-public/pr/7643~31..upstream-public/pr/7643 mbedtls-2.28..1fca541a5fd16abb5b1b4ceb15c46d4df62f5297`. Differences (other than changes in the context that don't matter):

* Missing 6f804693e5 "Fix wrong target names in the Makefile in tests/data_files" — justified (pkcs7 not present in 2.28)
* Changed 460b6cf0ba "Add server5-der*crt generate command" — has 6561f7d8ab "server5-der*.crt: Simplify the size calculation" combined into it
* Missing 0a7108f32d "Extend the validity period of pkcs7-rsa-sha*.crt to 10 years" — justified (pkcs7 not present in 2.28)
* Missing cd378f2ffe "Update pkcs7 files" — justified (pkcs7 not present in 2.28)
* Changed commit message and file location in 5e2f312195 "Update cert macros in tests/src/certs.c" becoming 14f59bfca8 "Update cert macros in library/certs.c" — justified (different file location)
* Missing 6561f7d8ab "server5-der*.crt: Simplify the size calculation" — combined above
* Missing b8b9cd4a79 "Mark all_intermediate as .SECONDARY" — why?

I can't think of other things that need to change between development and 2.28. So ok for the rebase part. (I don't know why not take "Mark all_intermediate as .SECONDARY" but it doesn't really matter.)